pkg/controller/user-cluster-controller-manager/resources/resources/node-local-dns/deamonset.go

```diff
@@ -30,6 +30,8 @@ import (
 	"k8s.io/utils/pointer"
 )
 
+const CacheAddress = "169.254.20.10"
```

Comment:
This is called `NodeLocalDNSCacheAddress = "169.254.20.10"` in  pkg/resources/machinecontroller/deployment.go
Let us use the same name here too.